src/components/MainLayout/CircleNav.tsx

```diff
@@ -38,13 +41,13 @@ export const CircleNav = () => {
         [paths.epoch(circleId), paths.team(circleId), paths.give(circleId)],
       ],
       [paths.members(circleId), 'Members'],
-      [paths.map(circleId), 'Map'],
     ];
-
+    if (showGiveInfoQuery.data?.showGiveInfo || myUser.isCircleAdmin)
```

Comment:
@sshmm @wingfeatherwave this prevents a non-admin user from accessing the map at all if `showGiveInfo` is disabled, but that means they can't even look at past epochs... is there a reason i shouldn't revert this?